mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/115 - Objectives load timing

Closes #115 

To test:
- Run one scenario and make sure objectives match scenario
- Exit and run a different scenario and make sure objectives are correct
